<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>768</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_camera">
     <attribute name="title">
      <string>深度相机</string>
     </attribute>
     <widget class="QLabel" name="clolorframe">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>480</width>
        <height>320</height>
       </rect>
      </property>
      <property name="text">
       <string>clolorframe</string>
      </property>
     </widget>
     <widget class="QLabel" name="irFrame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>480</width>
        <height>320</height>
       </rect>
      </property>
      <property name="text">
       <string>irFrame</string>
      </property>
     </widget>
     <widget class="QLabel" name="depthFrame">
      <property name="geometry">
       <rect>
        <x>512</x>
        <y>20</y>
        <width>480</width>
        <height>320</height>
       </rect>
      </property>
      <property name="text">
       <string>depthFrame</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>470</y>
        <width>160</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="savePicButton">
         <property name="text">
          <string>savePic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveCloudButton">
         <property name="text">
          <string>saveCloud</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_continue">
     <attribute name="title">
      <string>连续配准 </string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_import">
     <attribute name="title">
      <string>单帧配准及模型导入</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>20</y>
        <width>198</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="locale">
          <locale language="Chinese" country="China"/>
         </property>
         <property name="text">
          <string>stl,obj,ply格式CAD模型文件多视角采样</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>分辨率</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit">
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>细分数</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="selectCADFileButton">
         <property name="text">
          <string>选择模型文件并开始采样</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>200</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>410</y>
        <width>220</width>
        <height>20</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>230</y>
        <width>198</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="locale">
          <locale language="Chinese" country="China"/>
         </property>
         <property name="text">
          <string>pcd,ply点云生成点对特征</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>点云采样栅格</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="text">
            <string>0.04</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>距离离散步长</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="text">
            <string>0.04</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>角度离散步长</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_5">
           <property name="text">
            <string>30</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="selectPointCloudFileButton">
         <property name="text">
          <string>选择点云生成PPF</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QOpenGLWidget" name="openGLWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>800</width>
        <height>640</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>0</y>
        <width>20</width>
        <height>720</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>440</y>
        <width>198</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="locale">
          <locale language="Chinese" country="China"/>
         </property>
         <property name="text">
          <string>选择场景点云进行匹配</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>点云采样栅格</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_6">
           <property name="text">
            <string>0.04</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_7">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>场景点比例</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="selectPointCloudFileButton_2">
         <property name="text">
          <string>选择点云进行匹配</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
